README.md

```diff
@@ -95,31 +97,33 @@ In some cases (such as recent Ubuntu systems), it may be necessary to overcome f
+- `execute` and `composeAndExecute` add configured event logs into the `public.event_logs` table.
+- `extractDiffs` pulls state diffs into the `public.storage_diff` table.
 
-### Data transformation
-Data transformation uses the raw data that has been synced into Postgres to filter out and apply transformations to specific data of interest. Since there are different types of data that may be useful for observing smart contracts, it follows that there are different ways to transform this data. We've started by categorizing this into Generic and Custom transformers:
+### Transforming
+Data transformation uses the raw data that has been synced into Postgres to filter out and apply transformations to specific data of interest.
+Since there are different types of data that may be useful for observing smart contracts, it follows that there are different ways to transform this data.
+We've started by categorizing this into Generic and Custom transformers:
 
-- Generic Contract Transformer: Generic contract transformation can be done using a built-in command, `contractWatcher`, which transforms contract events provided the contract's ABI is available. It also provides some state variable coverage by automating polling of public methods, with some restrictions. `contractWatcher` is described further [here](documentation/generic-transformer.md).
-
-- Custom Transformers: In many cases custom transformers will need to be written to provide more comprehensive coverage of contract data. In this case we have provided the `compose`, `execute`, and `composeAndExecute` commands for running custom transformers from external repositories. Documentation on how to write, build and run custom transformers as Go plugins can be found [here](documentation/custom-transformers.md).
-
-### Exposing the data
-
-[Postgraphile](https://www.graphile.org/postgraphile/) is used to expose GraphQL endpoints for our database schemas, this is described in detail [here](documentation/postgraphile.md).
+- Generic Contract Transformer: Generic contract transformation can be done using a built-in command, `contractWatcher`, which transforms contract events provided the contract's ABI is available.
+`contractWatcher` is described further [here](documentation/generic-transformer.md).
 
+- Custom Transformers: In many cases custom transformers will need to be written to provide more comprehensive coverage of contract data.
+In this case we have provided the `compose`, `execute`, and `composeAndExecute` commands for running custom transformers from external repositories.
+Documentation on how to write, build and run custom transformers as Go plugins can be found [here](documentation/custom-transformers.md).
 
 ### Tests
 - Replace the empty `ipcPath` in the `environments/testing.toml` with a path to a full node's eth_jsonrpc endpoint (e.g. local geth node ipc path or infura url)
```

Comment:
Only needed for integrationtest - probably worth noting.

Reply:
At this point `make test` still fails if you don't either do this or pass a `CLIENT_IPCPATH` env var. I don't think that really should be the case, but maybe it makes sense to address that in a new story?

Reply:
👍 I didn't think that was the case.